floris/simulation/farm.py

```diff
@@ -13,6 +13,7 @@
 from __future__ import annotations
 
 import copy
+import json
```

Comment:
It may be safer to use `import yaml` here. This works, but since json is a subset of yaml there may be yaml features that aren't handled.

Reply:
That makes sense, I'll test this out to try and get a clean swap.